x-pack/plugins/security_solution/cypress/integration/timelines/notes_tab.spec.ts

```diff
@@ -41,8 +41,8 @@ describe('Timeline notes tab', () => {
           .click({ force: true })
```

Comment:
Seems that it's still have some problem with line 40, could we try changing `openTimelineById` in x-pack/plugins/security_solution/cypress/tasks/timeline.ts Line 207 to use pipe and see if that helps?
```
export const openTimelineById = (timelineId: string) => {
  cy.get(TIMELINE_TITLE_BY_ID(timelineId)).pipe(($el) => $el.trigger('click'));
};
```